### ISSUE TITLE
Is verifyPhoneNumber missing ?

### ISSUE THREAD

**Issue by vegetablesalad** [arnesson/cordova-plugin-firebase#639]
README shows example with "verifyPhoneNumber", but there is no such function in source-code.

**Comment by vegetablesalad** [comment]
sorry cancel that, I was looking at wrong source

**Comment by vegetablesalad** [comment]
These are all the functions that are available to me:
![image](https://user-images.githubusercontent.com/2006277/38680170-4e31acc4-3e6e-11e8-876c-4ed9fce9d03f.png)

There are two files **firebase.js** and **firebase-browser.js** I'm not quite sure, but it seems that I get the browser one. I'm testing this on physical android device!

**Comment by vegetablesalad** [comment]
I had to manually add this function to **firebase.js**

```
exports.verifyPhoneNumber = function (number, timeout, success, error) {
    let args=[];
    if (typeof number === 'string') {
        args.push(number);
    }
    args.push(timeout);
    exec(success, error, "FirebasePlugin", "verifyPhoneNumber", args);
};
```

**Comment by cedrickbae** [comment]
Yes I also noticed this. getVerificationID() is also missing.

**Comment by cedrickbae** [comment]
Did you manage to make your code work. Mine says Class not found everytime I call verifyPhoneNumber

**Comment by cedrickbae** [comment]
Where did you add the code, in node_modules/cordova-plugin-firebase or in plugins/cordova-plugin-firebase? I added it in both but the still the function throws class not found.

**Comment by vegetablesalad** [comment]
Add it in `plugins/cordova-plugin-firebase`. And most importantly you need to remove and add platform, otherwise the new code wont be copied from `plugins` to your platform directory. Or another option is to add the code manually inside platform.

**Comment by vegetablesalad** [comment]
`plugins/` directory is just where code is held until you add a `platform`. Only time plugin is copied to `platform` is when you install new plugin(the plugin is copied) or add new platform(all plugins are copied)

**Comment by soumak77** [comment]
fixed in version 1.0.2 which is being published now
